www/class/centreonStatistics.class.php

```diff
@@ -137,6 +138,138 @@ public function getPlatformTimezone()
+                        ? count($customConfiguration['blacklist_client_addresses']) : 0,
+                    'pattern_matching_login' => ($customConfiguration['pattern_matching_login'] ? true : false),
+                    'pattern_replace_login' => ($customConfiguration['pattern_replace_login'] ? true : false),
+                ];
+            } elseif ($row['type'] === 'openid') {
+                $authenticationConditions = $customConfiguration['authentication_conditions'];
+                $groupsMapping = $customConfiguration['groups_mapping'];
+                $rolesMapping = $customConfiguration['roles_mapping'];
+                $data['openid'] = [
+                    'is_forced' => $row['is_forced'] ? true : false,
+                    'authenticationConditions' => [
+                        'is_enabled' => $authenticationConditions['is_enabled'] ? true : false,
+                        'trusted_client_addresses'
+                            => isset($authenticationConditions['trusted_client_addresses'])
+                            ? count($authenticationConditions['trusted_client_addresses']) : 0,
+                        'blacklist_client_addresses'
+                            => isset($authenticationConditions['blacklist_client_addresses'])
+                            ? count($authenticationConditions['blacklist_client_addresses']) : 0,
+                        'authorized_values' => isset($authenticationConditions['authorized_values'])
+                            ? count($authenticationConditions['authorized_values']) : 0
+                    ],
+                    'groups_mapping' => [
+                        'is_enabled' => $groupsMapping['is_enabled'] ? true : false,
+                        'relations' => $cgRelations
+                    ],
+                    'roles_mapping' => [
+                        'is_enabled' => $rolesMapping['is_enabled'] ? true : false,
+                        'apply_only_first_role' => $rolesMapping['apply_only_first_role'] ? true : false,
+                        'relations' => $aclRelations
+                    ],
+                    'introspection_token_endpoint' => ($customConfiguration['introspection_token_endpoint']
+                        ? true : false),
+                    'userinfo_endpoint' => ($customConfiguration['userinfo_endpoint'] ? true : false),
+                    'endsession_endpoint' => ($customConfiguration['endsession_endpoint'] ? true : false),
+                    'connection_scopes' => isset($customConfiguration['connection_scopes'])
+                        ? count($customConfiguration['connection_scopes']) : 0,
+                    'authentication_type' => $customConfiguration['authentication_type'],
+                    'verify_peer' => ($customConfiguration['verify_peer'] ? true : false),
+                    'auto_import' => ($customConfiguration['auto_import'] ? true : false)
```

Comment:
Same as cast 👉️ propagate in other places of this commit if you change these two lines.
```suggestion
                    'verify_peer' => (bool)$customConfiguration['verify_peer'],
                    'auto_import' => (bool)$customConfiguration['auto_import']
```